packages/edit-site/src/components/page-patterns/dataviews-pattern-actions.js

```diff
@@ -104,7 +104,7 @@ export const deleteAction = {
 	},
 	hideModalHeader: true,
 	supportsBulk: true,
-	RenderModal: ( { items, closeModal, onPerform } ) => {
+	RenderModal: ( { items, closeModal, onActionPerformed } ) => {
```

Comment:
If I'm not wrong, this was a bug too. I see that all actions everywhere in dataviews use `onActionPerformed` prop and not `onPerform`. This is also something that I found thanks to the types.

Reply:
Yes, it seems it was a bug.